pkg/step/feature/derivation.go

```diff
@@ -386,7 +386,7 @@ func InferFeatureColumns(trainStmt *ir.TrainStmt, dataSource string) error {
 
 	// set back trainStmt.Features in the order of select
 	for _, target := range columnTargets {
-		targetFeatureColumnMap := fcMap[target]
+		targetColumnMap := fcMap[target]
```

Comment:
The original name `targetFeatureColumnMap` refers to Tensorflow feature column, maybe it's better than `ColumnMap`?

Reply:
I see. I will follow your suggestion and change it back